recipes/mTEDx/ASR/CTC/hparams/train_wav2vec.yaml

```diff
@@ -0,0 +1,175 @@
+# ################################
+# Model: wav2vec2 + DNN + CTC
+# Authors: Titouan Parcollet 2021
+#          Mohamed Anwar 2022
+# ################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1234
+__set_seed: !!python/object/apply:torch.manual_seed [!ref <seed>]
+output_folder: !PLACEHOLDER
+wer_file: !ref <output_folder>/wer.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# URL for the biggest LeBenchmark wav2vec french.
+wav2vec2_hub: LeBenchmark/wav2vec2-FR-7K-large
+
+# Data files
+data_folder: !PLACEHOLDER
+langs:
+   - fr
+remove_punc_cap: True # remove punctuation & capitalization from text
+
+train_json: !ref <data_folder>/train_fr.json
+valid_json: !ref <data_folder>/valid_fr.json
+test_json:  !ref <data_folder>/test_fr.json
+
+# We remove train utterances longer than 10s
+avoid_if_longer_than: 20.0
+test_lang: fr
+
+# Training parameters
+number_of_epochs: 30
+number_of_ctc_epochs: 15
+lr: 1.0
+lr_wav2vec: 0.0001
+ctc_weight: 0.3
+sorting: descending # ascending, descending
+auto_mix_prec: True
+sample_rate: 16000
+max_grad_norm: -1
+
+batch_size: 4
+batch_size_valid: 2 #for valid & test
+
+train_dataloader_opts:
+    batch_size: !ref <batch_size>
+    num_workers: 4
+
+valid_dataloader_opts:
+    batch_size: !ref <batch_size_valid>
+    num_workers: 4
+
+test_dataloader_opts:
+    batch_size: !ref <batch_size_valid>
+    num_workers: 4
+
+# BPE parameters
+token_type: char  # ["unigram", "bpe", "char"]
+character_coverage: 1.0
+
+# Model parameters
+activation: !name:torch.nn.LeakyReLU
+wav2vec_output_dim: 1024
+dnn_neurons: 1024
+freeze_wav2vec: False
+
+# Outputs
+output_neurons: 161  # BPE size, index(blank/eos/bos) = 0
+
+# Decoding parameters
+# Be sure that the bos and eos index match with the BPEs ones
+blank_index: 0
+bos_index: 1
+eos_index: 2
+min_decode_ratio: 0.0
+max_decode_ratio: 1.0
+beam_size: 80
+eos_threshold: 1.5
+using_max_attn_shift: True
+max_attn_shift: 140
+ctc_weight_decode: 0.0
+temperature: 1.50
+
+#
+# Functions and classes
+#
+epoch_counter: !new:speechbrain.utils.epoch_loop.EpochCounter
+    limit: !ref <number_of_epochs>
+
+enc: !new:speechbrain.nnet.containers.Sequential
+    input_shape: [null, null, !ref <wav2vec_output_dim>]
+    linear1: !name:speechbrain.nnet.linear.Linear
+        n_neurons: 1024
+        bias: True
+    bn1: !name:speechbrain.nnet.normalization.BatchNorm1d
+    activation: !new:torch.nn.LeakyReLU
+    drop: !new:torch.nn.Dropout
+        p: 0.15
+    linear2: !name:speechbrain.nnet.linear.Linear
+        n_neurons: 1024
+        bias: True
+    bn2: !name:speechbrain.nnet.normalization.BatchNorm1d
+    activation2: !new:torch.nn.LeakyReLU
+    drop2: !new:torch.nn.Dropout
+        p: 0.15
+    linear3: !name:speechbrain.nnet.linear.Linear
+        n_neurons: 1024
```

Comment:
does one of the `n_neuons` parameter relate to the above `dnn_neurons` ?
(just seeing the same value so I match patterns and ask)

Reply:
fixed it, thanks for the heads-up!